### DISCUSSION TITLE
Example configurations for code and phase biases estimation for PPP-AR

### DISCUSSION THREAD

**Discussion by whu-dyf** [GeoscienceAustralia/ginan#80] [Q&A]
Dear ginan team
I have a query regarding bias estimation configurations: are there any example configurations that describe how to jointly estimate code and phase biases? 

I modified the pod_example.yaml, but the output of .BIA file is empty.

**Comment by ronaldmaj** [comment]
Hi [whu-dyf](https://github.com/whu-dyf),

Thanks for your question, we recently moved to version 3 of the Ginan software and are still working through updating our various example configuration files - the estimation of phase biases being one of them.

Unfortunately, that one is not ready for public release yet, but we are currently working on it. 
Once we have that ready, that should be available on our develop-weekly branch. I will let you know when that has occurred.

**Comment by whu-dyf** [comment]
Dear Ginan Team,

Thank you for your update regarding the transition to version 3 of the Ginan software. I noticed that Ginan has now been updated to version 3.1. I was wondering if the example configuration files, particularly for the estimation of phase biases, are available now. I would greatly appreciate it if you could provide access to these files, as they are essential for my work.

Thank you for your assistance.

Best regards
